packages/base/src/3dview/mainview.tsx

```diff
@@ -377,6 +399,8 @@ export class MainView extends React.Component<IProps, IStates> {
     this._renderer.setRenderTarget(null);
     this._renderer.clearDepth();
     this._renderer.render(this._scene, this._camera);
+    this._renderer.autoClear = false;
+    this._viewHelper.render(this._renderer);
```

Comment:
We should set `autoClear` to false only once when creating the renderer, in here though, we need to call `clear` and not only `clearDepth` to both clear the color and depth buffers

```suggestion
    this._renderer.clear();
    this._renderer.render(this._scene, this._camera);
    this._viewHelper.render(this._renderer);
```

Reply:
Thanks, merged in [`ffa3a5b`](https://github.com/jupytercad/JupyterCAD/pull/440/commits/ffa3a5bb701eb355685a2c6532669e72916df89f)